db/migrations/20230406025628_create_user_restriction_table.js

```diff
@@ -0,0 +1,18 @@
+const { baseDown } = require('../utils')
+
+const table = 'user_restriction'
+
+exports.up = async (knex) => {
+  await knex('entity_type').insert({ table })
+  await knex.schema.createTable(table, (t) => {
+    t.bigIncrements('id').primary()
+    t.bigInteger('user_id').unsigned().notNullable()
+    t.enu('type', ['articleHottest', 'articleNewest']).notNullable()
+    t.timestamp('created_at').defaultTo(knex.fn.now())
```

Comment:
shall we need a `state` (`archived`, `active`) field so we don't need to delete the record?

Reply:
the `state` col will make db operation a little more complex, and according the production requirement by now,  we do not need keep the history of the restriction operation?

Reply:
some `state` kind of soft delete is still better; maybe some time later we will want to analyze